torch/csrc/lazy/backend/backend_device.cpp

```diff
@@ -53,5 +54,16 @@ c10::Device backendDeviceToAtenDevice(const BackendDevice& device) {
   return c10::Device(at::kLazy, device.ordinal());
 }
 
+c10::optional<BackendDevice> GetBackendDevice(const at::Tensor& tensor) {
+  if (auto lt = TryGetLtcTensor(tensor)) {
+    return lt.GetDevice();
+  }
+  return c10::nullopt;
+}
+
+c10::optional<BackendDevice> GetBackendDevice() {
+  return c10::nullopt;
```

Comment:
What is the point of this function?

Reply:
nevermind i see the comment in the header.

Reply:
Great!